apps/docs/spec/common-client-libs-sections.json

```diff
@@ -402,6 +402,7 @@
         "slug": "auth-error-codes",
         "type": "markdown",
         "excludes": [
+          "reference_javascript_v2",
```

Comment:
Opted not to exclude entirely just yet as [Python uses error codes](https://supabase.com/docs/reference/python/auth-error-codes) and the implementation might differ. Can remove after checking with library owner

Reply:
See comment above, the error codes table should be shared, we just need to move the intro text into tabs I think